versions/q-/qt5-webengine.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "afcb720c703b3ae92374d971e8a4047f589430c0",
+      "git-tree": "0a233d6658dbd5c32ebf0327447442972c64681a",
+      "version": "5.15.5",
+      "port-version": 1
+    },
+    {
+      "git-tree": "56874d7856a67a290d1a839c3711437a7ba1985b",
```

```suggestion
      "git-tree": "afcb720c703b3ae92374d971e8a4047f589430c0",
```
`qt5-webengine` is adding new port-version records, so the old records do not need to be changed.